site/app/templates/EditPronounsForm.twig

```diff
@@ -0,0 +1,40 @@
+{% extends 'generic/Popup.twig' %}
+{% block popup_id %}edit-pronouns-form{% endblock %}
+{% block title %}Edit Pronouns{% endblock %}
+{% block body %}
+    <p class="std-margin">{{ "Pronouns are the part of speech used to refer to someone in the third person. We want to know how to respectfully refer to you!
+    Pronouns will only be visible to members of the teaching staff who have the instructor role." }}</p>
+    <p class="std-margin">
+        Your instructor and the system administrators reserve the right to remove
+        pronouns if they contain inappropriate or offensive language.
+    </p>
+    <div class="flex-col" id="user_settings_modal">
+        <input id="user-pronouns-csrf" type="hidden" name="csrf_token" value="{{ csrf_token }}" />
+        <h2>My Preferred Pronouns</h2>
+        <div class="edit_user_row">
+            <label for="user-pronouns-change">Preferred Pronouns:&emsp;</label>
+            <input type="text"
+                   name="user_pronouns_change"
+                   value="{{ user_pronouns }}"
+                   id="user-pronouns-change"
+                   list="default_pronouns_list"
+                   placeholder="Type Custom Or Select">
+                <datalist id="default_pronouns_list">
+                    <option value="He/Him"></option>
+                    <option value="She/Her"></option>
+                    <option value="They/Them"></option>
+                    <option value="Custom(Type Here)"></option>
```

Comment:
Just my opinion: "Custom" is descriptive enough and "Type Here" is excessively verbose.  Not necessarily a requested change, just my opinion.

Reply:
fixed